tests/integration/test_eoc_journal.py

```diff
@@ -485,7 +487,7 @@ def test_engagement_metrics_listed_in_student_view_when_enabled(self):
 
         element = self.get_element_for_course_engagement()
 
-        for key, value in default_engagement_metrics.iteritems():
+        for key, value in list(default_engagement_metrics.items()):
```

Comment:
I don't think a list conversion is needed here. 
```suggestion
        for key, value in default_engagement_metrics.items():
```